metagraph/src/cli/align.hpp

```diff
@@ -19,7 +18,7 @@ class Config;
 
 graph::align::DBGAlignerConfig initialize_aligner_config(const Config &config);
 
-template <class Graph = graph::DeBruijnGraph>
+template <class Graph, typename... AlignerArgs>
 std::unique_ptr<graph::align::IDBGAligner>
 build_aligner(const Graph &graph, const graph::align::DBGAlignerConfig &aligner_config);
```

Comment:
it will become simpler with `template <class Graph, typename Aligner>`, also the usage will be more straightforward because you'll see every time which aligner exactly is going to be used.